reconcile/test/test_vault_replication.py

```diff
@@ -0,0 +1,31 @@
+from unittest import TestCase
+
+import reconcile.vault_replication as integ
+
+
+class TestRunInteg(TestCase):
+    def test_policy_contais_path(self):
+        policy_paths = ["path1", "path2"]
+        path = "path1"
+        self.assertTrue(integ.policy_contains_path(path, policy_paths))
+
+    def test_policy_contais_path_false(self):
+        policy_paths = ["path2", "path3"]
+        path = "path1"
+        self.assertFalse(integ.policy_contains_path(path, policy_paths))
+
+    def test_check_invalid_paths_ko(self):
+        path_list = ["path1", "path3"]
+        policy_paths = ["path1", "path2"]
+        with self.assertRaises(integ.VaultInvalidPaths):
+            integ.check_invalid_paths(path_list, policy_paths)
+
+    def test_check_invalid_paths_ok(self):
+        path_list = ["path1", "path2"]
+        policy_paths = ["path1", "path2"]
+        integ.check_invalid_paths(path_list, policy_paths)
+
+    def test_list_invalid_paths(self):
+        path_list = ["path1", "path3"]
+        policy_paths = ["path1", "path2"]
+        self.assertEqual(integ.list_invalid_paths(path_list, policy_paths), ["path3"])
```

Comment:
any chance to test the rest of the integration? Maybe mocking vault client?

Reply:
To add a related question, what percentage of the new code is covered, and what are we targeting? A percentage of coverage isn't the end goal by any means, but it's a reasonable start.

Reply:
Added some more tests for the integration, now we are at 70% coverage or so, I'll add a few more tests for the placeholder replacement function that I lastly added